addons/portal/portal_data.xml

```diff
@@ -7,5 +7,47 @@
             <field name="is_portal" eval="True"/>
         </record>
 
+        <record id="welcome_mail_portal_user" model="mail.template">
+            <field name="name">Welcome New Portal User</field>
+            <field name="model_id" ref="portal.model_portal_wizard_user"/>
+            <field name="email_from">${(user.email or '')|safe}</field>
+            <field name="email_to">${object.partner_id.email}</field>
+            <field name="subject">Your Odoo account at ${user.company_id.name|safe}</field>
+            <field name="lang">${object.partner_id.lang}</field>
+            <field name="auto_delete" eval="True"/>
+            <field name="body_html"><![CDATA[
+            	<p>
+                    Dear ${'user' in ctx and ctx['user'].name or ''},
```

Comment:
Isn't it possible to do obj.partner_id and obj.partner_id.name ? As obj shoudl be a wizard user, having a partner_id link to the partner.